core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -133,6 +137,11 @@ public void render() {
             return;
         }
 
+        if(settingsShowing){
+            drawSettingsScreen();
```

Comment:
Settings screen seems to useless in the current version. Add with included functionality in version 1.1.